### ISSUE TITLE
Restore-CohesityVMwareVM -PoweredOn switch argument does not work as intended

### ISSUE THREAD

**Issue by viciousvex** [cohesity/cohesity-powershell-module#47]
Using the following command to restore a VM, the VM is automatically powered on, regardless of the switch setting (-PoweredOn:$false or leaving out -PoweredOn switch alltogether):

```
Restore-CohesityVMwareVM -TaskName $restorejob.JobName -JobId $restorejob.JobId -SourceId $sid -DisableNetwork -NewParentId $parentvcenter.ProtectionSource.Id -ResourcePoolId $resourcepools[0].Id -DatastoreId $datastores[2].Id -PoweredOn:$false

Restore-CohesityVMwareVM -TaskName $restorejob.JobName -JobId $restorejob.JobId -SourceId $sid -DisableNetwork -NewParentId $parentvcenter.ProtectionSource.Id -ResourcePoolId $resourcepools[0].Id -DatastoreId $datastores[2].Id
```

**Comment by sagnihotri-cohesity** [comment]
Fix in progress. Will be included in the next release.

**Comment by bijayprasadtiwari** [comment]
Merged the fix with the master. The fix will be available in the release immediate to v1.2.1
